React-Projects/job-portal/src/api.js

```diff
@@ -0,0 +1,125 @@
+import {
+  SIGNUP_URL,
+  LOGIN_URL,
+  POST_JOB_URL,
+  GET_JOB_URL,
+  GET_SINGLE_JOB_URL,
+} from "./Constants";
+
+export const registerUser = async (fullname, email, password) => {
+  const requestBody = {
+    name: fullname,
+    email: email,
+    password: password,
+  };
+
+  const response = await fetch(SIGNUP_URL, {
```

Comment:
@codal-kcolaco Please update this API call to axios.

Reply:
@abhi-codal Understood